File: app/src/main/java/forpdateam/ru/forpda/api/news/models/Material.java
package forpdateam.ru.forpda.api.news.models;

/**
 * Created by radiationx on 30.08.17.
 */

public class Material {
    private int id;
    private String title;
    private String imageUrl;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/java/forpdateam/ru/forpda/entity/remote/news/Material.kt
package forpdateam.ru.forpda.entity.remote.news

import ru.radiationx.coretypes.ArticleId

/**
 * Created by radiationx on 30.08.17.
 */

data class Material(
    val id: ArticleId,
    val title: String,
    val imageUrl: String,
)
